applicant-frontend/src/static.json

```diff
@@ -48,15 +48,15 @@
   },
   "SUMMARY_VIEW": {
     "WELCOME_MESSAGE": "Welcome back {{ name }}",
-    "WELCOME_DESCRIPTION": "In this page you will find your current progress, you have to complete all these steps. You can do it in the order you prefer. You can save your answers and submit them later, or submit them directly!",
+    "WELCOME_DESCRIPTION": "In this page, you will find your current progress, you have to complete all these steps. You can do it in the order you prefer. You can save your answers and submit them later, or submit them directly!",
     "ICONS_DESCRIPTIONS":
     {
       "TODO_ICON_DESCRIPTION": "means that you have not yet completed this step, you have to answer and submit it (the number is the index of the step)",
       "PENDING_ICON_DESCRIPTION": "means that your answers are waiting an approval from Social Hackers Academy, it would come soon",
       "REJECTED_ICON_DESCRIPTION": "means that unfortunately, your answer was rejected by Social Hackers Academy: you can find on your emails some comments about it. You have to submit it again",
       "VALIDATED_ICON_DESCRIPTION": "means that your answer was accepted by Social Hackers Academy, perfect!"
     },
-    "DEADLINE_TEXT": "You have to complete all these steps before the {{ deadline }}",
```

Comment:
No, the deadline is the one you choose for the process, and the text is this one.
![image](https://user-images.githubusercontent.com/27521221/61324064-d0a69800-a811-11e9-8206-93c089282ca6.png)